### PR TITLE
Platform checker

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

**Comment by stefanpenner** [comment]
@homu +r

**Comment by rwjblue** [comment]
@homu r+

File: lib/cli/cli.js [hunk 1]
```diff
@@ -53,12 +53,16 @@ CLI.prototype.run = function(environment) {
...
+    if (!platform.isValid && !this.testing) {
       if (platform.isDeprecated) {
-        this.ui.writeDeprecateLine('Node ' + process.version + ' is no longer supported by Ember CLI. Please update to a more recent version of Node');
+        this.ui.writeDeprecateLine('Node ' + process.version +
+                                   ' is no longer supported by Ember CLI. Please update to a more recent version of Node');
       }
+
       if (platform.isUntested) {
-        this.ui.writeWarnLine('Node ' + process.version + ' has currently not been tested against Ember CLI and may result in unexpected behaviour.');
+        var chalk = require('chalk');
```
Comment (rwjblue) [inline lib/cli/cli.js]: This is unused (and causing a JSHint failure).

**Comment by rwjblue** [comment]
Rebased and fixed JSHint issue.

**Comment by rwjblue** [comment]
@homu r+

**Pull request merged**